### PR TITLE
Misc Fix Vyper

### PR DESCRIPTION
**Pull request by yann300**
fix https://github.com/ethereum/remix-project/issues/2720

 - fix displaying the title in the status icon

### CODE REVIEW

**Review by Aniket-Engg — COMMENTED** [review]
I see error on no file selected and on remote compiler compilation.
Screenshot 2022-08-16 at 4 42 41 PM

**Comment by yann300** [comment]
The bad request is thrown when the compilation fails, there isn't much to do here.
For the `NoFileSelected`, we explicitly remove the log (it always throw when no file are selected)

File: apps/vyper/src/app/utils/remix-client.tsx [hunk 1]
```diff
@@ -14,34 +15,70 @@ export class RemixClient extends PluginClient {
...
     }
   }
 
+  async cloneVyperRepo() {
+    try {
+      // @ts-ignore
+      this.call('notification', 'toast', 'cloning Vyper repository...')
+      await this.call('manager', 'activatePlugin', 'dGitProvider')
+      // @ts-ignore
+      await this.call('dGitProvider', 'clone', { url: 'https://github.com/vyperlang/vyper', token: null }, 'vyper-lang')
```
Comment (Aniket-Engg) [inline apps/vyper/src/app/utils/remix-client.tsx:46]: You mentioned `Vyper` at other places but creating workspace with `vyper-lang` name

**Review by Aniket-Engg — APPROVED** [review]

File: apps/vyper/src/app/utils/remix-client.tsx (hunk 1 above)
Comment (yann300) [reply apps/vyper/src/app/utils/remix-client.tsx:46]: that's the issue I was discussing with with @bunsenstraat. we can't have nested folder with same name...

Comment (Aniket-Engg) [reply apps/vyper/src/app/utils/remix-client.tsx:46]: ok

**Comment by PatrickAlphaC** [comment]
Awesome! Looking forward to the release.

**Pull request merged**